packages/eslint-config-migme/preact.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  extends: [
+    './base',
+    './rules/strict',
+    './rules/preact',
```

Comment:
Updated. Now `rules/preact.js` is importing from `rules/react.js` first, then modify only setting. so that we can ensure the consistency between rules for React and Preact.